quickstarts/microsoft.web/app-service-managed-certificate-apex/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "domainName": {
+      "type": "string",
+      "metadata": {
+        "description": "Type your domain name. Ex.: contoso.com"
+      }
+    },
+    "websiteName": {
+      "type": "string",
+      "metadata": {
+        "description": "Web App name that will host the Custom Domain. This name should be unique in Azure."
+      }
+    }
+  },
+  "variables": {
+    "certificateName": "[toLower(concat(parameters('domainName')))]",
+    "sku": "s1",
+    "workerSize": "0",
+    "AppServicePlanName": "[concat(toLower(parameters('websiteName')), 'ASP')]",
+  },
+  "resources": [
+    {
+      "apiVersion": "2018-02-01",
+      "name": "[variables('AppServicePlanName')]",
+      "type": "Microsoft.Web/serverfarms",
+      "location": "[resourceGroup().location]",
+      "sku": {
+        "name": "[variables('sku')]",
+        "capacity": "[variables('workerSize')]"
+      },
+      "properties": {
+        "name": "[variables('AppServicePlanName')]"
+      }
+    },
+    {
+      "apiVersion": "2018-02-01",
+      "name": "[parameters('websiteName')]",
+      "type": "Microsoft.Web/sites",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "serverFarmId": "[variables('AppServicePlanName')]"
+      },
+      "dependsOn": [
```

Comment:
move this above properties

See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties